packages/eslint-plugin-template/tests/rules/i18n/cases.ts

```diff
@@ -181,15 +190,15 @@ export const invalid = [
       'should fail if `i18n` attribute is missing on element containing bound text',
     annotatedSource: `
       <div>
-        <span>-{{data_from_backend}}</span>
```

Comment:
I've moved this test to the valid section, as it should be valid since there's no letter here.